File: LAB-1/Pr1.c
#include <stdio.h>

void main() {
    int n;
    printf("Enter Number\n");
    scanf("%d", &n);

    if(n > 0) {
        printf("Positive\n");
    } else if(n < 0) {
        printf("Negative\n");
    } else {
        printf("Zero\n");
    }

}

File: LAB-5/Pr1.java
import java.util.Scanner;

class Pr1 {
    public static void main(String[] args) {
        System.out.println("Enter a number:");
        Scanner scanner = new Scanner(System.in);
        int num = scanner.nextInt();
        String result = (num > 0)? "Positive" : (num < 0) ? "Negative" : "Zero";
        System.out.println(result);
    }    
}
